### ISSUE TITLE
Unable to submit attributes for launch with agent for newman

### ISSUE THREAD

**Issue by gitpingli** [reportportal/agent-js-postman#31]
Hi guys,
CLI for newman to upload launch is as follow, but i cannot set attributes for the launch.
Could you help to support it? Or is there any way i could set attributes for the uploaded launch?
$ newman run https://postman-echo.com/status/200 \
    -r @reportportal/reportportal \
    --reporter-@reportportal/reportportal-debug=true \
    --reporter-@reportportal/reportportal-endpoint=http://your-instance.com:8080/api/v1 \
    --reporter-@reportportal/reportportal-token=00000000-0000-0000-0000-000000000000 \
    --reporter-@reportportal/reportportal-launch=LAUNCH_NAME \
    --reporter-@reportportal/reportportal-project=PROJECT_NAME \
    -x

**Comment by gitpingli** [comment]
I found out how agent postman to support attributes for report portal:
- Add variable tags in postman collection variables.
- agent-js-postman extracts variable 'tags' from the whole postman collection variables. And use it to set attributes for launch in report portal

However attributes may be changed per launch. Such as we want to set build version for each launch. Then it is better to pass the attribute dynamically instead of adding it to collecitons.
Below patch works from local side. Does anyone help to check whether we could add it to this repo?

```
diff --git a/lib/index.js b/lib/index.js
index e6af199..cee721d 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -91,7 +91,7 @@ module.exports = function (emitter, options, collectionRunOptions) {
 
         launchObj = client.startLaunch({
             description: description,
-            tags: getTags(collectionRunOptions.collection.variables)
+            tags: getTags(options.tags)
         });
 
         launchObj.promise.catch(errorHandler);
@@ -106,7 +106,7 @@ module.exports = function (emitter, options, collectionRunOptions) {
         const testObj = client.startTestItem({
             name: o.item.name,
             type: 'TEST',
-            tags: getTags(collectionRunOptions.collection.variables)
+            tags: getTags(options.tags)
         },
         launchObj.tempId);
 
@@ -131,7 +131,7 @@ module.exports = function (emitter, options, collectionRunOptions) {
             stepObj = client.startTestItem({
                 name: stepName,
                 type: 'STEP',
-                tags: getTags(collectionRunOptions.environment.values)
+                tags: getTags(options.tags)
             },
             launchObj.tempId,
             testObj.testId);
@@ -164,7 +164,7 @@ module.exports = function (emitter, options, collectionRunOptions) {
                 const stepObj = client.startTestItem({
                     name: stepName,
                     type: 'STEP',
-                    tags: getTags(collectionRunOptions.environment.values)
+                    tags: getTags(options.tags)
                 },
                 launchObj.tempId,
                 testObj.testId);
@@ -379,10 +379,19 @@ module.exports = function (emitter, options, collectionRunOptions) {
      * @returns {Array} Array of tags.
      */
     function getTags (variables) {
-        const tags = _.find(variables.members, {
+        /*const tags = _.find(variables.members, {
             key: 'tags'
+        });*/
+        var attributes = [];
+		variables.split(',').forEach((keyPair) => {
+            if (keyPair.includes(":")) {
+                attributes.push({"key": keyPair.split(":")[0], "value": keyPair.split(":")[1]});
+            } else {
+                attributes.push({"value": keyPair});
+            }
         });
 
-        return tags && tags.value ? tags.value.split(';') : [];
+        //return tags && tags.value ? tags.value.split(';') : [];
+        return attributes;
     }
 };
```

**Comment by AmsterGet** [comment]
@KatsiarynaTatarynovich What do you think about it? Maybe it makes sense to include such feature in version `5.0.1`?

**Comment by KatsiarynaTatarynovich** [comment]
Hi, @gitpingli agent-js-postman has been released recently. In the newest version, there is an opportunity to submit attributes for launch. Please see the example https://github.com/reportportal/agent-js-postman#programmatically
Please tell me, is this what you wanted?

**Comment by gitpingli** [comment]
Thanks @KatsiarynaTatarynovich .
How could i set attributes through newman CLI?

**Comment by KatsiarynaTatarynovich** [comment]
@gitpingli unfortunately, you can't set attributes through Newman CLI, only programmatically. As soon as possible new agent's version with the feature will be released that will allow you to work with the command line.

**Comment by AmsterGet** [comment]
Hi there!
We've already released a fix in the version [5.0.1](https://github.com/reportportal/agent-js-postman/releases/tag/v5.0.1).
I'll close this issue but feel free to open it again if you have any other questions on the topic.
